lib/checks/checks.go

```diff
@@ -47,7 +47,36 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var log = logrus.WithField(trace.Component, "checks")
+const (
+	failedISCSIProbeMessage = "Found conflicting systemd service: %v. " +
+		"If this service is present on the host it will interfere " +
+		"with OpenEBS enabled applications running in Gravity." +
+		"Please stop and mask this service and try again."
+)
+
+var (
+	fmtISCSICheckFailedMsg = func(unitName string) string {
+		return fmt.Sprintf(failedISCSIProbeMessage, unitName)
+	}
+
+	// DefaultProcessesToCheck is the default list of processes to verify that are not running on the host.
+	// This list should be passed to the DefaultProcessChecker function.
+	DefaultProcessesToCheck = []string{
```

Comment:
Isn't this the same list as in satellite? Why not just use the one from there?

Reply:
This list was moved from Satellite to here because it is Gravity specific. Satellite still has a default list but the obvious Gravity specific processes are removed from that list and the list is parametrized. This is needed because the goal of the Satellite project is to be independent of gravity. After the new changes it will be possible to use the functionality provided by Satellite as a general purpose process list checker.

Reply:
Satellite is not really an independent project from gravity at this point in time, and satellite would need a significant investment to work independently / with non gravity clusters.